crates/starknet-devnet-core/src/state/mod.rs

```diff
@@ -71,13 +71,6 @@ impl CommittedClassStorage {
         self.committed.extend(self.staging.drain());
         diff
     }
-
-    /// Skips the staging phase
-    fn insert_and_commit(&mut self, class_hash: ClassHash, contract_class: ContractClass) {
```

Comment:
Can you explain why this can be removed now?

Reply:
It was only used in predeclaration; instead of staging (i.e. inserting) and immediately committing one predeclared class at a time, we can first insert all of them and just commit once (this is done in `Starknet::new`).

Reply:
ok, I also checked `StateDiff::generate` and it's more clear now